docs/config/index.rst

```diff
@@ -205,20 +225,22 @@ To see what configuration parameters are defined for a given ``conf``::
 
 You can also iterate through ``conf`` in a dictionary-like fashion::
 
-    >>> [key for key in conf.keys()]
+    >>> list(conf.keys())
     ['auto_download',
      'auto_max_age',
      ...,
      'ietf_leap_second_auto_url']
-    >>> [cfgitem for cfgitem in conf.values()]
+    >>> list(conf.values())
     [<ConfigItem: name='auto_download' value=True at ...>,
      <ConfigItem: name='auto_max_age' value=30.0 at ...>,
      ...,
      <ConfigItem: name='ietf_leap_second_auto_url' value=...>]
     >>> for (key, cfgitem) in conf.items():
     ...     if key == 'auto_max_age':
     ...         print(f'{cfgitem.description} Value is {cfgitem()}')
-    Maximum age (days) of predictive data before auto-downloading. Default is 30. Value is 30.0
+    Maximum age (days) of predictive data before auto-downloading. See "Auto
+    refresh behavior" in astropy.utils.iers documentation for details.Default
```

Comment:
The lack of whitespace after period is a bug if it is indeed what is printed on screen.
```suggestion
    refresh behavior" in astropy.utils.iers documentation for details. Default
```

Reply:
There really is no whitespace: https://github.com/astropy/astropy/blob/23404eeb2caeea315cea89b6cb8b9b2fee4f13be/astropy/utils/iers/iers.py#L121-L125
Should I fix it here or would a separate pull request be more appropriate?

Reply:
Personally, I think it is trivial enough to tack on here, but since this is IERS, let's ask @mhvk or @eteq .

Reply:
Definitely just go ahead and fix it here! 

Aside: thanks for yet another great set of clean-ups!